Applications/timeline_command_dispatcher.c

```diff
@@ -259,85 +262,69 @@ static PH_ACK drop_tl_cmd_at_(int line_no, cycle_t time)
 
 CCP_EXEC_STS Cmd_TLCD_SET_SOE_FLAG(const CommonCmdPacket* packet)
 {
-  const unsigned char* param = CCP_get_param_head(packet);
-  uint8_t line_no;
-  uint8_t flag;
-
-  // パラメータ読み出し&内容確認
-  line_no = param[0];
+  TL_ID line_no = (TL_ID)CCP_get_param_from_packet(packet, 0, uint8_t);
+  uint8_t flag = CCP_get_param_from_packet(packet, 1, uint8_t);
 
   if (line_no >= TL_ID_MAX)
   {
     // 指定されたライン番号が存在しない場合は異常判定。
     return CCP_EXEC_ILLEGAL_PARAMETER;
   }
 
-  flag = param[1];
-
   if ((flag != 0) && (flag != 1))
   {
     // フラグ内容が0/1でないなら異常判定。
     return CCP_EXEC_ILLEGAL_PARAMETER;
   }
 
   // 異常時実行中断フラグを設定
-  timeline_command_dispatcher_[line_no].stop_on_error = flag;
+  timeline_command_dispatcher_.dispatcher[line_no].stop_on_error = flag;
   return CCP_EXEC_SUCCESS;
 }
 
 CCP_EXEC_STS Cmd_TLCD_SET_LOUT_FLAG(const CommonCmdPacket* packet)
 {
-  const unsigned char* param = CCP_get_param_head(packet);
-  uint8_t line_no;
-  uint8_t flag;
-
-  // パラメータ読み出し&値チェック。
-  line_no = param[0];
+  TL_ID line_no = (TL_ID)CCP_get_param_from_packet(packet, 0, uint8_t);
+  uint8_t flag = CCP_get_param_from_packet(packet, 1, uint8_t);
 
   if (line_no >= TL_ID_MAX)
   {
     // 存在しないライン番号が指定された場合は異常判定。
     return CCP_EXEC_ILLEGAL_PARAMETER;
   }
 
-  flag = param[1];
-
   if ((flag != 0) && (flag != 1))
   {
     // フラグ情報が0/1でない場合は異常判定。
     return CCP_EXEC_ILLEGAL_PARAMETER;
   }
 
   // コマンド実行フラグを設定。
-  timeline_command_dispatcher_[line_no].lockout = flag;
+  timeline_command_dispatcher_.dispatcher[line_no].lockout = flag;
   return CCP_EXEC_SUCCESS;
 }
 
 CCP_EXEC_STS Cmd_TLCD_SET_LINE_NO_FOR_TIMELINE_TLM(const CommonCmdPacket* packet)
```

Comment:
[IMO] Cmd_TLCD_SET_PAGE_FOR_TLM と合わせる？

```suggestion
CCP_EXEC_STS Cmd_TLCD_SET_LINE_NO_FOR_TLM(const CommonCmdPacket* packet)
```

Reply:
あと，

```
CCP_EXEC_STS Cmd_TLCD_SET_LINE_NO_FOR_TIMELINE_TLM(const CommonCmdPacket* packet);
CCP_EXEC_STS Cmd_TLCD_DEPLOY_BLOCK(const CommonCmdPacket* packet);
CCP_EXEC_STS Cmd_TLCD_SET_PAGE_FOR_TLM(const CommonCmdPacket* packet);
CCP_EXEC_STS Cmd_TLCD_CLEAR_ERR_LOG(const CommonCmdPacket* packet);
```
の順番変えたいね． set for tlmは最後で良い気がする（cmd dbも）